CustomTabBar/CustomTabBar.swift

```diff
@@ -69,12 +69,24 @@ public protocol CustomTabBarDelegate {
         }
     }
     
+    public var titleFont: UIFont = UIFont.systemFontOfSize(16) {
+        didSet {
+            updateTabs()
+        }
+    }
+    
     @IBInspectable public var lightBackground: Bool = true {
         didSet {
             updateBackground()
         }
     }
     
+    @IBInspectable public var darkBackground: Bool = false {
```

Comment:
That's what I thought originally. But I could never change the background color of the tabbar to anything but the light or dark color.  . . This was the only way I could ever change it. I had to turn off both light and dark to be able to change it. I'm definitely open to suggestions on this.